index.html

```diff
@@ -96,16 +96,15 @@ <h3 class="head_tertiary feature-name">Technologies</h3>
           <h2 class="header_secondary">What are we doing</h2>
           <ul class="list list-primery">
             <li class="feature">
-              <img src="./images/img1.jpg" alt="computer" width="360" />

```

Comment:
I suggest not using html attribute `width=` to set size of the image, is better to control it from css